app/assets/javascripts/models/user.js

```diff
@@ -1,3 +1,15 @@
 module.exports = Backbone.Model.extend({
-  name: 'user'
+  name: 'user',
+  initialize: function() {
```

Comment:
I think this would solve the problem https://github.com/Codeminer42/cm42-central/pull/106#discussion_r99101087